### PR TITLE
save setting to QSetting

### PR DESCRIPTION
**Pull request by meomancer**
add setting utillities

### CODE REVIEW

File: cadasta/api/login.py [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
+__date__ = '16/12/16'
+__copyright__ = 'Copyright 2016, Cadasta'
 
 
 class Login(NetworkMixin):
     def __init__(self, domain, username, password, on_finished=None):
         """
-        Constructor
+        Constructor.
         :param username:
```
Comment (ismailsunni) [inline cadasta/api/login.py]: Fill the description and the type of each params

File: cadasta/common/setting.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+default_domain = 'https://demo.cadasta.org/'
+
+
+def set_setting(key, value):
+    """
+    Set setting.
+    :param key: unique key for setting
+    :type key: QString
+    :param value: value to be saved
```
Comment (ismailsunni) [inline cadasta/common/setting.py]: New line before new param description

File: cadasta/common/setting.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    :type value: QVariant
+
+    """
+    settings = QSettings()
+    settings.setValue(key, value)
+
+
+def delete_setting(key):
+    """
+    delete setting.
```
Comment (ismailsunni) [inline cadasta/common/setting.py]: Put the short description on the first line, with Sentence case.

**Review by ismailsunni — COMMENTED** [review]
Please update

File: cadasta/common/setting.py [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+def delete_setting(key):
+    """ Delete setting from QSettings.
+
+    :param key: unique key for setting.
+    :type key: QString
+
+    """
+    settings = QSettings()
+    settings.remove(key)
```
Comment (Gustry) [inline cadasta/common/setting.py:43]: You have to add a prefix such as 'cadasta/' to the key. QSettings is shared among QGIS and every plugins so you have to add prefix.
In QGIS Desktop, then the QGIS preferences and then `advanced` you can see all the keys in the QSettings.

**Review by Gustry — COMMENTED** [review]
Can you please this ?

**Pull request merged**